AsyncDisplayKit/Details/ASRangeControllerBeta.mm

```diff
@@ -275,15 +283,22 @@ - (void)dataController:(ASDataController *)dataController didDeleteNodes:(NSArra
   });
 }
 
-- (void)dataController:(ASDataController *)dataController didInsertSections:(NSArray *)sections atIndexSet:(NSIndexSet *)indexSet withAnimationOptions:(ASDataControllerAnimationOptions)animationOptions
+- (void)dataController:(ASDataController *)dataController didInsertSectionsAtIndexSet:(NSIndexSet *)indexSet withAnimationOptions:(ASDataControllerAnimationOptions)animationOptions
 {
-  ASDisplayNodeAssert(sections.count == indexSet.count, @"Invalid sections");
```

Comment:
I eleminated the section parameter on this call. therefore no need to check length equality anymore.